lib/index.js

```diff
@@ -708,7 +708,15 @@ function _connect(options, callback) {
       server: config.connectOptions
     };
   }
-
+  if(config.username && config.password) {
+    connectOptions.auth = {
+      user: config.username,
+      password: config.password
+    };
```

Comment:
This is modifying the `config.connectOptions` ... which seems like a mistake. Are we missing a shallow copy?

Reply:
good call here. I'll make sure connect only recieves a shallow copy: https://github.com/digitalbazaar/bedrock-mongodb/pull/44/commits/48b470b5c1e1aef440042834be8a8a0ce733c8b2